Compile.ps1

```diff
@@ -9,6 +9,21 @@ $sync = [Hashtable]::Synchronized(@{})
 $sync.PSScriptRoot = $PSScriptRoot
 $sync.configs = @{}
 
+function Update-Progress {
+    param (
+        [Parameter(Mandatory, position=0)]
+        [string]$StatusMessage,
+
+	[Parameter(Mandatory, position=1)]
```

Comment:
```suggestion
	[Parameter(Mandatory, position=1)]
	[ValidateRange(0,100)]
```
This would make sure that nobody can fatfinger the percent in the future

Reply:
Added it in commit 78178b0fba5bc5647aca66e458b2f1bfa8e2e4d4
Thanks for the suggestion @Marterich 😄